web/src/lib/components/Auction.svelte

```diff
@@ -115,19 +115,37 @@
                     </div>
                 </div>
                 <div class="lg:w-1/2">
-                    {#if auction.bids.length}
```

Comment:
Wanted to fix the formatting, but I figured you are still working on this file, so I better not cause conflicts. But please try to keep the code tidy.